src/main.cpp

```diff
@@ -2064,7 +2049,7 @@ int main(int arg_count, char const **arg_ptr) {
 		#endif
 
 		#ifdef GIT_SHA
-		gb_printf("-%s", GIT_SHA);
+		gb_printf(":%s", GIT_SHA);
```

Comment:
Why the change?

Reply:
It just looked aesthetically wrong. Making it `:` make it not look like a date if the hash began with two numbers

Reply:
Hmm, not something I would see but a fair reason